### PR TITLE
Sebaszm/combined streamer

### PR DESCRIPTION
**Pull request by sebaszm**
Adds posibility to pick up and configure multiple streamers from the menuconfig.

### CODE REVIEW

File: package/aamp/aamp.mk [hunk 1]
```diff
@@ -9,7 +9,7 @@ AAMP_SITE_METHOD = git
 AAMP_SITE = https://github.com/rdkcmf/rdk-aamp
 AAMP_INSTALL_STAGING = YES
 
-AAMP_DEPENDENCIES = libcurl libdash libxml2 cjson aampabr gst1-plugins-base wpeframework
+AAMP_DEPENDENCIES = libcurl libdash libxml2 cjson aampabr
```
Comment (HaseenaSainul) [inline package/aamp/aamp.mk]: @sebaszm gst1-plugins-base and wpeframework are added to build these package before aamp in clean build. So does this work in clean build with out these ?

File: package/wpe/wpeframework-plugins/Config.in [hunk 2]
```diff
@@ -427,57 +427,155 @@ menuconfig BR2_PACKAGE_WPEFRAMEWORK_STREAMER
...
+      Enable broadcast player
+
+if BR2_PACKAGE_WPEFRAMEWORK_STREAMER_BROADCAST
 
 choice
-    bool "DBS Options"
-    default BR2_PACKAGE_WPEFRAMEWORK_STREAMER_DVB
-    help
-      Choose the DBS flavor.
+    prompt "Broadcast modus"
```
Comment (HaseenaSainul) [inline package/wpe/wpeframework-plugins/Config.in]: Is it modus or modems ?

Comment (sebaszm) [reply package/wpe/wpeframework-plugins/Config.in]: modus :)

File: package/aamp/aamp.mk (hunk 1 above)
Comment (sebaszm) [reply package/aamp/aamp.mk]: I'm not sure a this moment, will double check... but why would AAMP depend on WPEFramework?

Comment (HaseenaSainul) [reply package/aamp/aamp.mk]: It has ocdm header dependencies from WEPFramerwork

Comment (sebaszm) [reply package/aamp/aamp.mk]: You're right, I'll bring the dependecies for the time being back, though I don't feel this is a proper solution as we end up with circular dependencies.

Comment (HaseenaSainul) [reply package/aamp/aamp.mk]: Ok, which one is causing the circular dependency ?

Comment (sebaszm) [reply package/aamp/aamp.mk]: It seems that after rebasing the branch, buildroot no longer complains about circular dependencies. Nevermind, then!

**Pull request merged**